riva/clients/nmt/streaming_s2t_client.cc

```diff
@@ -218,7 +218,7 @@ StreamingS2TClient::DoStreamingFromFile(
   while (true) {
     while (NumActiveStreams() < (uint32_t)num_parallel_requests && all_wav_i < all_wav_max) {
       std::unique_ptr<Stream> stream(new Stream(all_wav_repeated[all_wav_i], all_wav_i));
-      StartNewStream(std::move(stream));
+      StartNewStream(std::move(stream), all_wav_repeated[all_wav_i]->filename);
```

Comment:
do we need to pass the filename through all these functions? is it possible to move to similar way as ASR where we are writing the file from client_call.cc which has access to the filename? In S2T as well, we get PostProcessResults which can call a ClientCall function to write the file.

Reply:
Looking into this